src/stylesheets/screens/lazy-loader.scss

```diff
@@ -1,31 +1,33 @@
 .skeleton-header {
```

Comment:
The filename must be alway match the class name so that it's easy to find. So here it should be `skeleton`? `lazy-loader` seems entirely unrelated to skeleton 😆  In addition, now both `skeleton-header` and `skeleton-details` are global class name while they should be namespace as `skeleton__header` and `skeleton__details`. Or in this case, both header and details seem like two components then create two files for each.

Reply:
To make things more consistent and not confusing I changed everything `skeleton` to `loader`. And also made separate stylesheet files for `loader-header` and `loader-details`(previously `skeleton-header` and `skeleton-details`). Updated in [ebc775c](https://github.com/rafayet-monon/surveyor/pull/13/commits/ebc775ca675f30bb3cf159b206e4c47e3b6be748)

Reply:
A small update made in [46261fd](https://github.com/rafayet-monon/surveyor/pull/13/commits/46261fd3c8960be1b9f2eada091395864d4a4b45) as I didn't delete some unnecessary styles from `loader-header.scss`